src/MarDyn.cpp

```diff
@@ -132,8 +132,18 @@ int run_unit_tests(const Values &options, const std::vector<std::string> &args)
  */
 int main(int argc, char** argv) {
 #ifdef ENABLE_MPI
+#if defined(_OPENMP)
```

Comment:
any reason not to merge the two?
```suggestion
#if defined(ENABLE_MPI) && defined(_OPENMP)
```

Reply:
Completely correct. Disregard this one.